### DISCUSSION TITLE
Is a Web build possible?

### DISCUSSION THREAD

**Discussion by 1upbyte** [pear-devs/pear-desktop#2351] [Q&A]
As this is a desktop version of webapp with plugins, is it possible to build for the browser? Even if needs a backend server to run that would be find such as Node.

**Comment by ArjixWasTaken** [comment]
with A LOT of modifications it is doable
it would probably be easier to write a browser version from scratch (and copy over common code)

**Reply by ArjixWasTaken** [reply]
+not all plugins would work

**Reply by ArjixWasTaken** [reply]
I'll give this a look, no promises though.
